src/Run/Executor.cs

```diff
@@ -25,24 +24,15 @@ public Executor()
         {
             SettingParameters = new Dictionary<string, string>();
             CommandParameters = new Dictionary<string, Dictionary<string, string>>();
-            string executorDirectory = Path.GetDirectoryName(typeof(Executor).GetTypeInfo().Assembly.Location);
-            configFilePath = Path.GetFullPath(Path.Combine(executorDirectory, @".."));
+            configFilePath = Directory.GetCurrentDirectory();
```

Comment:
Just to make sure we don't cause a bunch of breaks I would suggest staging this, by keeping the old behavior as well as adding the new behavior. If `..\config.json` exists use it but if not look in the current directory.